lib/Page.js

```diff
@@ -694,6 +694,43 @@ class Page extends EventEmitter {
   }
 
   /**
+   * @param {string} selector
+   * @param {string|string[]} value
+   */
+  async select(selector, value) {
+    const handle = await this.$(selector);
+    console.assert(handle, 'No node found for selector: ' + selector);
+
+    await this.evaluate((element, value) => {
+      if (element.nodeName.toLowerCase() !== 'select')
+        return false;
```

Comment:
On line 702 you throw if there's no element matching selector.
In order to be consistent, you should also throw if there's an element that is not a `<select>`.